packages/libevent/libevent.0.9.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "ygrek@autistici.org"
+license: "BSD-3-Clause"
+authors: [
+  "Maas-Maarten Zeeman <mmzeeman@xs4all.nl>"
+  "ygrek <ygrek@autistici.org>"
+]
+homepage: "https://github.com/ygrek/ocaml-libevent"
+dev-repo: "git+https://github.com/ygrek/ocaml-libevent.git"
+bug-reports: "https://github.com/ygrek/ocaml-libevent/issues"
+tags: ["org:ygrek"]
+build: [
+  [make "depend"]
+  [make]
+  [make "test"] {with-test & os != "macos"}
+  [make "doc"] {with-doc}
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml" {>= "4.02.0"}
+  "ocamlfind" {build}
+  "base-unix"
+  "base-bytes" {with-test}
+  "ounit" {with-test}
+  "conf-libevent" {build}
+]
+synopsis: "OCaml wrapper for the libevent API"
```

Comment:
```suggestion
conflicts: [ "ocaml-option-bytecode-only" ]
synopsis: "OCaml wrapper for the libevent API"
```
I cannot see the error right now, the CI has restarted on its own, but from @avsm comment I believe this is the fix

Reply:
I’m a bit surprised as we are able to compile this lib with byte code binaries and use it in things like utop. Also the previous version didn’t have this restriction and the changes in the code have been very minimal

Reply:
I think that is because the CI did not have a bytecode only compiler at that point. The failure is now accessible again:
```

#=== ERROR while compiling libevent.0.9.0 =====================================#
# context              2.2.0~beta3~dev | linux/x86_32 | ocaml-base-compiler.5.1.1 | pinned(https://ygrek.org/p/release/ocaml-libevent/ocaml-libevent-0.9.0.tar.gz)
# path                 ~/.opam/5.1/.opam-switch/build/libevent.0.9.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build make
# exit-code            2
# env-file             ~/.opam/log/libevent-7-c4eb3c.env
# output-file          ~/.opam/log/libevent-7-c4eb3c.out
### output ###
# ocamlfind ocamlc -package unix -c -ccopt " -Wall " event_stubs.c
# ocamlmklib -oc mloevent event_stubs.o -levent 
# ocamlfind ocamlc -package unix -c  libevent.mli
# ocamlfind ocamlc -package unix -c  -nolabels libevent.ml
# ocamlmklib -o liboevent libevent.cmo -oc mloevent -levent 
# ocamlfind ocamlopt -package unix -c  -nolabels libevent.ml
# ocamlfind: Not supported in your configuration: ocamlopt
# make: *** [Makefile:95: libevent.cmx] Error 2
```
You compile unconditionally both the bytecode and native versions, if no native compiler is available at all then this fails (the conflict is with system that only have bytecode compilers)